dev-requirements.txt

```diff
@@ -0,0 +1,7 @@
+absl-py=1.2.0
```

Comment:
Might be worth addin `-r requirements.txt` to this. So we don't need dupes between "reqs required to run" and "reqs required for dev".